Specification/4._CAN_bus_transport_layer.md

```diff
@@ -19,43 +19,39 @@ UAVCAN defines the following types of transfers:
+* Multi-frame transfer - a transfer that has its payload distributed over multiple CAN frames.
 
 The following properties are common to all types of transfer:
 
 Property        | Description
 ----------------|------------------------------------------------------------------------------------------------------
 Payload         | The serialized data structure
-Data type ID    | This identifier indicates how the data structure should be interpreted
-Priority        | See below
-Transfer ID     | An integer value that allows receiving nodes to distinguish this transfer from others
+Data type ID    | An identifier that indicates how the data structure should be interpreted
+Priority        | An positive integer value that defines the message urgency (higher numbers indicate reduced priority). Higher priority transfers can delay transmission of loser priority transfers. 
+Transfer ID     | An integer value that allows receiving nodes to distinguish this transfer from all others
 
-The transfer priority is an integer number that defines the urgency of the data contained in the transfer.
-Numerically lower priority values indicate higher urgency, and numerically higher values indicate lower urgency.
-Transfers of higher priority (i.e. with numerically lower priority value) can delay transmission of transfers
-whose priority are lower (i.e. numerically higher).
 
 ### Message broadcasting
 
-A broadcasted message is carried by a single message transfer that contains the serialized message.
-A broadcasted message has the following properties:
+A broadcast message is carried by a single-frame transfer that contains the serialized message.
+A broadcast message has the following properties:
 
 Property        | Description
 ----------------|------------------------------------------------------------------------------------------------------
 Payload         | The serialized message
-Data type ID    | This identifier indicates how the message should be interpreted
+Data type ID    | An identifier that indicates how the message should be interpreted
 Source node ID  | Node ID of the node that has transmitted the transfer
 Priority        | See above
 Transfer ID     | See above
 
 In order to broadcast a message, the broadcasting node must have a node ID that is unique within the network.
-An exception is applied to *anonymous message transfers*.
+An exception is applied to *anonymous message broadcasts*.
 
 #### Anonymous message broadcasting
 
-Anonymous message is a message that originates from a node that does not have a node ID.
-This sort of message transfer is useful for dynamic node ID allocation purposes - a high-level concept that
-will be explained in the following parts of the specification.
+An anonymous message is a transfer that originates from a node that does not have a node ID.
+This sort of message transfer is useful for *dynamic node ID allocation* (a high-level concept that
```

Comment:
I understand dynamic node ID allocation, but I don't understand how anonymous message broadcasting helps up with dynamic node ID allocation, as stated here.

Reply:
It is explained in chapter 6. Do you think the reference should be made explicit? Or perhaps the whole sentence about the feature being useful for dynamic node ID allocation should be removed?

Reply:
If we keep it here, then yes, I would make an explicit reference (and I'd leave that to you). 
IMO we should remove it, and instead add a back-reference from chapter 6 - but I would leave that for you as a post-step.